README.md

```diff
@@ -9,15 +9,15 @@ Licenses are looked up in the dataset and if a vehicle is found, that data is di
 
 ## Development
 
-### If you just want to run the app
+We use the [`just` command runner](https://github.com/casey/just), you'll need to install it before running any steps below.
 
-Build the image with `docker build -t <tag> .`. Run it with `docker run -p 5000:5000 <tag>`.
+### Build and run the app
+
+Build the image with `just build` and run it using `just up`.
+The server will be available at port `3030`.
```

Comment:
Should we add a small note about how to use `IS_PROD`? Is it just `IS_PROD=true just up`?

Reply:
Great idea!